docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,125 @@
+---
+uid: admin-hardware-selection
+title: Hardware Selection
+---
+
+# Selecting Appropriate Hardware
+
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (ex. Hardware Acceleration).
+
+## TL;DR
+
+For a Jellyfin server, we recommend the following:
+
+### Normal Server From Scratch
+
+- CPU: 4 Threads
+  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
+  - Please Avoid Rapsberry PIs and other low powered SBCs.
+  - Please Avoid Intel Core M / Y series CPUs.
+- RAM: 8GB or more
+- Storage: SSD for Jellyfin files and transcoding cache.
+- Graphics: Intel > Nvidia > AMD > Apple.
+  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
+  - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
+  - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.
+  - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.
+
+### Low Power Applications
+
+For low power applications, we recommend Intel 12th gen or newer Atom CPUs with integrated graphics.
+
+### Upgrading Existing System for use as a Server
+
+We recommend Intel ARC GPUs for upgrading existing systems. If Intel ARC GPUs aren't available, we recommend Nvidia GTX 16 / RTX 20 or newer, excluding the GTX 1650.
+
+## Detailed Guide
+
+### CPU
+
+The CPU will be responsible for many tasks, Notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs.
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
```

Comment:
I don't think this is true if source is 4k HDR

Reply:
That's fair, I will add some clarification